src/main/java/Bank/State/InterestState.java

```diff
@@ -4,4 +4,10 @@
 
     void calc();
 
+    Interest getInvestment();
```

Comment:
To mają być liczone odsetki
- investment -> lokata
- interest -> odsetki

Więc jak dla mnie tutaj będzie tylko funkcja licząca